tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -25,13 +25,27 @@ use virt_autotest::utils qw(is_kvm_host is_xen_host);
 
 sub run {
 
+    #enable ssh root access for openSUSE TW in sol console
+    #root access is not permitted by default in TW, see bsc#1173067#c2
+    select_console 'sol', await_console => 1;
+    send_key 'ret' if check_screen('sol-console-wait-typing-ret');
+    if (check_screen('text-login')) {
+        enter_cmd "root";
+        assert_screen "password-prompt";
+        type_password;
+        send_key('ret');
+    }
+    assert_screen "text-logged-in-root";
+    enter_cmd("echo \"PermitRootLogin yes\" > /etc/ssh/sshd_config");
+    enter_cmd("systemctl restart sshd");
+
     #skip TW host installation and directly login if you'd like to run test on an SUT with TW installed
     if (get_var('IPMI_DO_NOT_RESTART_HOST')) {
         select_console 'sol', await_console => 0;
         send_key 'ret';
     }
 
-    use_ssh_serial_console unless check_screen('text-logged-in-root');
+    use_ssh_serial_console;
```

Comment:
Can we just use api here, like select_console('root-ssh' or someting) ?  I do not know why we need to stick with use_ssh_serial_console

Reply:
yes. select_console('root-ssh') is enough here.